index.js

```diff
@@ -55,12 +68,18 @@ module.exports = function (mode) {
       }
       files = statusFiles;
 
-      if (mode === 'D') {
-        // Deleted files. Make into vinyl files
-        files.map(makeVinylFile).forEach(function (file) {
-          stream.push(file);
-        });
-      }
+      // Deleted files. Make into vinyl files
+      _.chain(files)
```

Comment:
I'm also thinking this could be written in a simpler way:

```js
files.reduce(function(acc, file) {
  if (file.mode !== 'D') return acc;
  return acc.concat(makeVinylFile(file.path));
}, []).forEach(function (file) {
  stream.push(file);
});
```

or even just

```js
files.forEach(function(acc, file) {
  if (file.mode !== 'D') return;
 stream.push(makeVinylFile(file.path));
});
```